js/src/mesh.js

```diff
@@ -40,7 +40,8 @@ var MeshView = widgets.WidgetView.extend( {
             fragmentShader: require('../glsl/mesh-fragment.glsl'),
             polygonOffset: true,
             polygonOffsetFactor: 1, // positive value pushes polygon further away, so wireframes will render properly (z-buffer issues)
-            polygonOffsetUnits: 1
+            polygonOffsetUnits: 1,
+            visible: this.model.get("visible")
```

Comment:
This would need to be copied to all the other materials. Otherwise I would still be visible when you do texture rendering for instance.